### DISCUSSION TITLE
Sincronización 780/G4 apple watch

### DISCUSSION THREAD

**Discussion by Manueladelaserna** [NightscoutFoundation/xDrip#3232] [Q&A]
Debo tener un error en la configuración de Nightcout que no permite la actualización de datos. El objetivo es que el reloj de mi hija "apple watch" lleve a la nube la información para que followers podamos tener su CGM

**Comment by Navid200** [comment]
We can help with xDrip, not Apple watch.
If you want to upload from xDrip, on Android, to Nightscout and it doesn't work, you can have a look at REST API and check the syntax to sort it out.
